api/src/main/java/org/pmiops/workbench/cdr/model/AnalysisResult.java

```diff
@@ -0,0 +1,240 @@
+        this.stratum4 = val;
+        return this;
+    }
+
+    @Column(name="stratum_4_name")
+    public String getStratum4Name() {
+        return stratum4Name;
+    }
+    public void setStratum4Name(String stratum4Name) {
+        this.stratum4Name = stratum4Name;
+    }
+    public AnalysisResult stratum4Name(String val) {
+        this.stratum4Name = val;
+        return this;
+    }
+
+    @Column(name="stratum_5")
+    public String getStratum5() {
+        return stratum5;
+    }
+    public void setStratum5(String stratum5) {
+        this.stratum5 = stratum5;
+    }
+    public AnalysisResult stratum5(String val) {
+        this.stratum5 = val;
+        return this;
+    }
+
+    @Column(name="stratum_5_name")
+    public String getStratum5Name() {
+        return stratum5Name;
+    }
+    public void setStratum5Name(String stratum5Name) {
+        this.stratum5Name = stratum5Name;
+    }
+    public AnalysisResult stratum5Name(String val) {
+        this.stratum5Name = val;
+        return this;
+    }
+
+    @Column(name="count_value")
+    public Long getCountValue() {
+        return countValue;
+    }
+    public void setCountValue(Long countValue) {
+        this.countValue = countValue;
+    }
+    public AnalysisResult countValue(Long val) {
+        this.countValue = val;
+        return this;
+    }
+
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        AnalysisResult that = (AnalysisResult) o;
+
+        if (id != null ? !id.equals(that.id) : that.id != null) return false;
+        if (analysisId != null ? !analysisId.equals(that.analysisId) : that.analysisId != null) return false;
+        if (stratum1 != null ? !stratum1.equals(that.stratum1) : that.stratum1 != null) return false;
+        if (stratum1Name != null ? !stratum1Name.equals(that.stratum1Name) : that.stratum1Name != null) return false;
+        if (stratum2 != null ? !stratum2.equals(that.stratum2) : that.stratum2 != null) return false;
+        if (stratum2Name != null ? !stratum2Name.equals(that.stratum2Name) : that.stratum2Name != null) return false;
+        if (stratum3 != null ? !stratum3.equals(that.stratum3) : that.stratum3 != null) return false;
+        if (stratum3Name != null ? !stratum3Name.equals(that.stratum3Name) : that.stratum3Name != null) return false;
+        if (stratum4 != null ? !stratum4.equals(that.stratum4) : that.stratum4 != null) return false;
+        if (stratum4Name != null ? !stratum4Name.equals(that.stratum4Name) : that.stratum4Name != null) return false;
+        if (stratum5 != null ? !stratum5.equals(that.stratum5) : that.stratum5 != null) return false;
+        if (stratum5Name != null ? !stratum5Name.equals(that.stratum5Name) : that.stratum5Name != null) return false;
+        return countValue != null ? countValue.equals(that.countValue) : that.countValue == null;
```

Comment:
Can we remove id from the equals method since it really means database equality instead of object equality? Can we use java.util.Objects here also. For Example:

```
if (this == o) return true;
     if (o == null || getClass() != o.getClass()) return false;
          AnalysisResult that = (AnalysisResult) o;
          return Objects.equals(analysisId, that.analysisId) && 
               Objects.equals(stratum1, that.stratum1) && 
               Objects.equals(stratum1Name, that.stratum1Name) &&
               Objects.equals(stratum2, that.stratum2) &&
               Objects.equals(stratum2Name, that.stratum2Name) &&
               Objects.equals(stratum3, that.stratum3) &&
               Objects.equals(stratum3Name, that.stratum3Name) &&  
               Objects.equals(stratum4, that.stratum4) &&
               Objects.equals(stratum4Name, that.stratum4Name) &&
               Objects.equals(stratum5, that.stratum5) &&  
               Objects.equals(stratum5Name, that.stratum5Name) &&
               Objects.equals(countValue, that.countValue);
```

Reply:
Yes.